handler/src/test/java/io/netty/handler/ssl/OpenSslRenegotiateTest.java

```diff
@@ -21,6 +21,8 @@
 
     @Override
     public void testRenegotiateServer() throws Throwable {
+        // BoringSSL does not support renegotiate by purpose.
+        Assume.assumeFalse("BoringSSL".equals(OpenSsl.versionString()));
```

Comment:
I'm just curious why versionString returns 'BoringSSL'. Shouldn't it be some version numbers or at least a string that contains the version number?